plugins/avg.js

```diff
@@ -121,9 +121,9 @@ exports.hook_data_post = function (next, connection) {
                     break;
                 case 'scan': {
                     const elapsed = Date.now() - start_time;
-                    connection.loginfo(plugin, 'time=' + elapsed + 'ms ' +
-                                    'code=' + code + ' ' +
-                                    'response="' + response.join(' ') + '"');
+                    connection.loginfo(plugin, `time= ${elapsed} ms\ 
+                    code= ${code}\ 
```

Comment:
and incorrect whitespace here.

Reply:
@msimerson I'm not sure what the best option is here? It's too long for one line.

Reply:
You can actually turn this into an object, the logger will turn it into key=value

Reply:
@KingNoosh Could I have an example? (Sorry, I'm fairly new to this feature and using it as a learning experience)

Reply:
You can have a long line (it's not a cardinal sin any longer) or convert it to key-value pairs as @KingNoosh alluded:

```js
connection.loginfo(plugin, {
    time: `${elapsed}ms`,
    code: code,
    response: `"${response.join(' ')}"`,
})
```